projects/tl-elements/src/lib/tl-card/tl-card.component.ts

```diff
@@ -21,6 +21,14 @@ export class TlCardComponent extends TamuAbstractBaseComponent {
   /** Designate the initial expanded/collapsed state. */
   @Input() startCollapsed: boolean;
 
+  /** Designate the actual expanded/collapsed state. */
+  @Input() collapsed: boolean;
+
+  /** Update the boolean when the element attribute changes. */
+  @HostBinding('attr.collapsed') get collapsedGetter(): string {
```

Comment:
We have a convention for getters and setters in TS to use `private _propName` , `get propName` , and `set propName`